src/main/java/com/google/graphgeckos/dashboard/datatypes/BuildStep.java

```diff
@@ -0,0 +1,153 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.graphgeckos.dashboard.datatypes;
+
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
+
+/**
+ * Contains the information for an individual step of a build.
+ */
+@Entity(name = "buildStep")
+public class BuildStep {
+
+  // Order of the step
+  @Field(name="step_number")
```

Comment:
this name `step_number` is in different style than the names below like `isFinished`. Consider updating this name to be consistent with the others